### PR TITLE
lib: add copy option to set manifest type

### LINKED ISSUE #161: add option to set manifest type while copying images
If we want to copy image from docker registry to oci registry, there should be an option to force manifest type so that we can make sure oci compatible blobs are transferred to oci registry. 

Similar to ```skopeo copy --format=oci```

### PR DESCRIPTION
**Pull request by shimish2**
Closes #161

### CODE REVIEW

**Comment by tych0** [comment]
Can you add a link showing what valid values of this string are; and perhaps a test?

File: lib/image.go [hunk 1]
```diff
@@ -113,6 +114,13 @@ func ImageCopy(opts ImageCopyOpts) error {
 		}
 	}
 
+	// Set ForceManifestMIMEType
+	// Supported manifest type :- https://github.com/anuvu/image/blob/master/manifest/manifest.go#L35
```
Comment (tych0) [inline lib/image.go:118]: Can you link to the upstream containers/image here?

Comment (tych0) [reply lib/image.go:118]: Ping?

Comment (shimish2) [reply lib/image.go]: Updated link, please review

**Pull request merged**